src/components/FullHierarchyTable/FullHierarchyTable.tsx

```diff
@@ -170,6 +307,56 @@ export const FullHierarchyTable: React.FunctionComponent<FullHierarchyTableAllPr
         }
     }
 
+    const dropDown = (filterableFieldsKey: string[], key: string) => {
+        if (filterableFieldsKey?.indexOf(key) > -1) {
+            return {
+                filterDropdown: (dropdownProps: FilterDropdownProps) =>
```

Comment:
This needs to be extracted into separated component

Reply:
make it in separate compopnet `customDropdown`